business/badgeCalculator.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+// Responsible for calculating what badge a component/definition has.
+// For now we are just going to go with the simple:
+// If license/copyright there it gets a 3. Only one of those it gets a 2. None, it gets a 1
+
+const _ = require('lodash');
+
+const scoreToUrl = {
+  1: 'https://img.shields.io/badge/Clearly%20Defined%20Score-1-red.svg',
```

Comment:
Feels odd to give component a positive score when nothing is defined.  Propose 0  when they have nothing

Reply:
I like this idea. It would be pretty jarring to get a score of 0.